visualization/app/codeCharta/ui/nodeContextMenu/flattenButtons/flattenButtons.component.html

```diff
@@ -0,0 +1,13 @@
+<button
+	mat-button
+	*ngIf="!codeMapNode?.isFlattened"
+	(click)="flattenNode()"
+	title="Flatten node and children-nodes, but keep an empty space"
+>
+	<img alt="flattenIcon" class="png-icon" src="../../../../assets/flatten.png" style="width: 17px; height: 17px" />
```

Comment:
isn't it better to have the Style in a separated File?

Reply:
Good point. I just kept previous implementation. I will improve that with last PR, to avoid rebasing / merging each